src/index.html

```diff
@@ -2,16 +2,176 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>
+      HTML Form
+    </title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      method="post" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="form-fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-field">
+          Surname:
+          <input 
+            type="text" 
+            id="surname"
+            name="surname" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text" 
+            id="name" 
+            name="name" 
+            minlength="3" 
+            maxlength="15" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            id="age" 
+            name="age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            type="date" 
+            id="birth" 
+            name="birth"
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            id="agree" 
+            name="agree"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            id="password" 
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-fieldset">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="form-fieldset">
```
You can add newlines between some blocks of code to make your code easier to read